### DISCUSSION TITLE
Windows cli : how to compare two folder

### DISCUSSION THREAD

**Discussion by CreaTech56** [qarmin/czkawka#1221] [Q&A] [answered]
I tried this :czkawka_cli.exe dup -d c:\FolderOne  c:\FolderTwo
I have this message : error: unexpected argument 'c:\FolderTwo' found
I try other synta but I don't find the good
Thanks for advance

**Comment by qarmin** [comment, marked as answer]
Maybe 
```
czkawka_cli.exe dup -d c:\FolderOne -d c:\FolderTwo
```
?

**Comment by CreaTech56** [comment]
Perfect
Thanks a lot
